lib/internal/process/esm_loader.js

```diff
@@ -43,6 +43,9 @@ const esmLoader = new ESMLoader();
 
 exports.esmLoader = esmLoader;
 
+exports.getFileSystem = () =>
+  esmLoader.fileSystem;
```

Comment:
This was required because there seems to be some dependency cycle between `esm_loader` and the modules used during the resolution, causing destructuring to fail.